### DISCUSSION TITLE
Thinking about updating doc to reference a compatible library

### DISCUSSION THREAD

**Discussion by piotr-yuxuan** [babashka/babashka#1614] [Q&A]
Hello, thanks for this great and enjoyable tool! 👋

It happens that [piotr-yuxuan/malli-cli](https://cljdoc.org/d/com.github.piotr-yuxuan/malli-cli) is compatible with babashka 🎉 

``` 
$ bb test

Running tests in #{"test"}

Testing piotr-yuxuan.malli-cli.domain.gnu-test

Testing piotr-yuxuan.malli-cli.domain.posix-test

Testing piotr-yuxuan.malli-cli-test

Ran 18 tests containing 135 assertions.
0 failures, 0 errors.
```

I've put the « babashka-compatible » badge in the readme, but now I'm thinking about adding it in bb doc. The update is ready to be merged: https://github.com/babashka/babashka/compare/master...piotr-yuxuan:babashka:master.

Thinking more, do you think there would be some need to turn malli-cli into a pod? It's not clear to me what it adds. `malli-cli` ships with no dependencies but obviously `malli` itself should be present when used. Happy to hear any remark or review to make `malli-cli` more usable and more helpful from within babashka 👍

Best,
Piotr

**Comment by piotr-yuxuan** [comment]
Relates to 🚀: https://github.com/babashka/babashka/pull/1574

**Comment by borkdude** [comment]
I commented on the PR. There is no need for a pod if the library already runs with bb.
